modules/contact-form/js/gutenblock.jsx

```diff
@@ -0,0 +1,630 @@
+/*global wp*/
+/** @jsx wp.element.createElement */
+/** @format */
+
+/**
+ * External dependencies
+ */
+import {
+	PanelBody,
+	TextControl,
+	TextareaControl,
+	CheckboxControl,
+	ToggleControl,
+	IconButton
+} from '@wordpress/components';
+
+import {
+	registerBlockType,
+	createBlock
+} from '@wordpress/blocks';
+
+import {
+	InnerBlocks,
+	InspectorControls
+} from '@wordpress/editor';
+
+import {
+	Component,
+	Fragment
+} from '@wordpress/element';
+
+import {
+	__
+} from '@wordpress/i18n';
+
+/**
+ * Components:
+ */
+
+class GrunionForm extends Component {
+	constructor( ...args ) {
+		super( ...args );
+		this.onChangeSubject = this.onChangeSubject.bind( this );
+		this.onChangeTo = this.onChangeTo.bind( this );
+		this.onChangeSubmit = this.onChangeSubmit.bind( this );
+	}
+
+	onChangeSubject( x ) {
+		this.props.setAttributes( { subject: x } );
+	}
+
+	onChangeTo( x ) {
+		this.props.setAttributes( { to: x } );
+	}
+
+	onChangeSubmit( x ) {
+		this.props.setAttributes( { submit_button_text: x } );
+	}
+
+	render() {
+		return (
+			<Fragment>
+				<InspectorControls>
+					<PanelBody title={ __( 'Submission Details', 'jetpack' ) }>
+						<TextControl
+							label={ __( 'What would you like the subject of the email to be?' ) }
+							value={ this.props.subject }
+							onChange={ this.onChangeSubject }
+						/>
+						<TextControl
+							label={ __( 'Which email address should we send the submissions to?' ) }
+							value={ this.props.to }
+							onChange={ this.onChangeTo }
+						/>
+						<TextControl
+							label={ __( 'What should the label on the form’s submit button say?' ) }
+							value={ this.props.submit_button_text }
+							placeholder={ __( 'Submit »' ) }
+							onChange={ this.onChangeSubmit }
+						/>
+					</PanelBody>
+				</InspectorControls>
+				<div className="grunion-form">
+					{this.props.children}
+					<TextControl
+						className="button button-primary button-default grunion-submit-button"
+						value={ this.props.submit_button_text }
+						placeholder={ __( 'Submit »' ) }
```

Comment:
`»` feels extraneous, could we leave it out?

Reply:
That's the legacy string that it already uses.  It didn't feel worth changing it and affecting translations.

Reply:
Looks like it's already included in translations so we'd be good to do the change:

https://translate.wordpress.org/projects/wp-plugins/jetpack/dev/en-gb/default?filters%5Bstatus%5D=either&filters%5Boriginal_id%5D=600163&filters%5Btranslation_id%5D=29847455